src/templates/TemplateData.ts

```diff
@@ -17,6 +17,7 @@ import { ScriptProjectCreatorBase } from '../commands/createNewProject/ScriptPro
 import { ProjectLanguage, ProjectRuntime, TemplateFilter } from '../constants';
 import { ext } from '../extensionVariables';
```

Comment:
I think you should still delete templates.zip. However, I _do_ think you should cache the backup templates so that you don't have to download/extract the next time